src/translator_ingest/ingests/alliance/alliance.py

```diff
@@ -0,0 +1,345 @@
+"""
+Alliance of Genome Resources ingest for mouse and rat data.
+
+This ingest processes gene-to-phenotype and gene-to-expression associations
+from the Alliance of Genome Resources, for mouse (NCBITaxon:10090) and
+rat (NCBITaxon:10116) only.
+
+All nodes are minimal placeholders (id + category only) intended to be merged
+with richer data from other sources.
+"""
+
+import uuid
+import koza
+from typing import List
+from loguru import logger
+from pathlib import Path
+import duckdb
+
+from biolink_model.datamodel.pydanticmodel_v2 import (
+    Gene,
+    GeneToPhenotypicFeatureAssociation,
+    GeneToExpressionSiteAssociation,
+    PhenotypicFeature,
+    AnatomicalEntity,
+    CellularComponent,
+    KnowledgeLevelEnum,
+    AgentTypeEnum,
+)
+
+# Source mapping for infores
+SOURCE_MAP = {
+    "FB": "infores:flybase",
+    "MGI": "infores:mgi",
+    "RGD": "infores:rgd",
+    "HGNC": "infores:rgd",
+    "SGD": "infores:sgd",
```

Comment:
Would we speed up a bit if we didn't have to make gene objects for excluded (at the moment) species?